server/app/views/applicant/ApplicantLayout.java

```diff
@@ -167,13 +183,28 @@ private ContainerTag<?> getLanguageForm(
   }
 
   private ATag branding() {
+    ImgTag cityImage;
+
+    if (maybeLogoUrl.isPresent()) {
+      cityImage = img().withSrc(maybeLogoUrl.get());
+    } else {
+      cityImage = this.layout.viewUtils.makeLocalImageTag("ChiefSeattle_Blue");
```

Comment:
**question:** So if a logo isn't set we use the Seattle logo? I feel like we should add no image if the logo isn't set? Or only use the Seattle logo if the entity is Seattle?